certora/specs/Immutability.spec

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// True when a delegate call has been placed.
+persistent ghost bool delegateCall;
+
+hook DELEGATECALL(uint g, address addr, uint argsOffset, uint argsLength, uint retOffset, uint retLength) uint rc {
+    delegateCall = true;
+}
+
+// Check that the contract is truly immutable.
```

Comment:
Same here, we should update that comment

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/90/commits/61d555bc6772a43f7c4e0eb27de11167599697eb.